draft-ietf-tls-tls13.md

```diff
@@ -1817,6 +1817,17 @@ the 0-RTT data without detection. If the client's ephemeral share
 is compromised and client authentication is used, then the attacker
 can impersonate the client on subsequent connections.
 
+It is up to the implementation and application layers above TLS to mitigate
+the impact of these weaknesses.  For example, to support strong client
+authentication and forward secrecy over 0-RTT connections, it is possible to
+issue a new ticket on the first 1-RTT connection, and on every resumed session
+after that.  Each co-located group of servers could ensure that tickets they
+issue are used only once by maintaining a local cache of ticket decryption
+keys.  An accurate clock on the client could ensure freshness of client
+signatures.  In addition it is a good idea to make all request idempotent,
```

Comment:
I think I left out an important detail that might be confusing.  The server needs to reject the CertificateVerify if the signed timestamp is too old, or too far in the future.  a 30 second to 1 minute window is common, I think, with OTPs.  This is not as strong as the typical 1-ish second window we get with a real proof of possession, but we're just suggesting techniques for mitigating weakness here.